app/views/idv/shared/_failure_to_proof_url.html.slim

```diff
@@ -0,0 +1,8 @@
+hr
+  .mb2.mt2
+    .right = link_to image_tag(asset_url('carat-right.svg'), size: '10'),
+      decorated_session.failure_to_proof_url, class: 'bold block btn-link text-decoration-none'
+    = link_to raw('Get help at <strong>' + decorated_session.sp_name + '</strong>'),
```

Comment:
You could probably use a content tag helper in the presenter to avoid have to call `raw` here.

Reply:
Also, we had an offline discussion about a conditional to check that `sp_name` and `failure_to_proof_url` are both present before rendering. That could probably got outside the partial here.